rm/rm-server/build.gradle

```diff
@@ -70,4 +70,7 @@ artifacts {
 }
 
 functionalTest.dependsOn rootProject.dist, dist
-clean.dependsOn rootProject.cleanDist
\ No newline at end of file
+clean.dependsOn rootProject.cleanDist
+
+task limitTheMaximumNumberOfNodes(type: maxNodes.ReplaceMaxNumberOfNodesTask) {
```

Comment:
It would be nice to have an example somewhere about how to invoke the task, and especially how to specify the maximum number of nodes that is desired.